docs/api/wrapper/attributes.md

```diff
@@ -13,3 +13,11 @@ import Foo from './Foo.vue'
 const wrapper = mount(Foo)
 expect(wrapper.attributes().id).toBe('foo')
 ```
+
```

Comment:
I think we should just move the assertion to the above example, no need to duplicate the set up.

eg:

```
import { mount } from '@vue/test-utils'
import Foo from './Foo.vue'	 import Foo from './Foo.vue'
 	 
const wrapper = mount(Foo)	const wrapper = mount(Foo)
expect(wrapper.attributes().id).toBe('foo')
expect(wrapper.attributes('id')).toBe('foo')
```

Reply:
Same for the rest of the docs pages